### ISSUE TITLE
Better batch design

### ISSUE THREAD

**Issue by siddharthab** [BioSTEAMDevelopmentGroup/biosteam#170]
Streams currently only support continuous flow as volume, mass or molar units per hour.

This leads to non-obvious unit designs when operating in batch mode. For example, a StirredTankReactor should be able to take its feed from a StorageTank and Pump combination to load much faster than what the continuous flow mode would allow. The current implementation of batch design for StirredTankReactor assumes that the continuous feed is being used to load one reactor at a time. This is inefficient utilization of each reactor, especially if you want to stagger fewer number of reactors, or if residence time in the reactors is very long.

**Comment by yoelcortes** [comment]
Good point! This not something I plan on taking care of any time soon, but it would be great to have as an option (to assume a given loading time). 

The following would need to be implemented:

add optional loading_time argument to size_batch method (and code).
add optional loading_time argument to BatchBioreactor/StirredTankReactor.
When loading_time is given, BatchBioreactor/StirredTankReactor could include an auxiliary surge tank (and pump) with the required surge time.

Thanks!

**Comment by yoelcortes** [comment]
If you would like to work on this, I would be happy to help through the process and review.

Thanks

**Comment by siddharthab** [comment]
Thanks! Handling just the case of StirredTankReactor seems straightforward. I may visit this if I find more time in the coming weeks.
